pkgs/development/tools/chefdk/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, bundlerEnv, bundlerUpdateScript, ruby_2_4, perl, autoconf }:
 
 bundlerEnv {
-  pname = "chef-dk";
+  name = "chef-dk-2.4.17";
```

Comment:
Apparently `bundlerEnv` supports using:

```nix
  pname = "chef-dk";
  version = "2.4.17";
```

Reply:
Then the build is still broken.

Reply:
Because, somehow it breaks if it uses basicEnv: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ruby-modules/bundler-env/default.nix#L38-L60